airflow-core/src/airflow/cli/cli_config.py

```diff
@@ -355,6 +355,13 @@ def string_lower_type(val):
 )
 
 # test_dag
+ARG_DAGFILE_PATH = Arg(
+    (
+        "-f",
+        "--dagfile-path",
+    ),
+    help="Path to the DAG file. Can be absolute or relative to current directory",
```

Comment:
```suggestion
    help="Path to the dag file. Can be absolute or relative to current directory",
```

Not sure whether we should use `dag` instead. In my memory servers me right, our latest conconsion is to update this usage in docs if possible